templates/world-map-2d/packages/client/node/deno.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@world-map-2d/node",
+  "version": "0.1.0",
+  "license": "MIT",
+  "exports": {
+    ".": "./src/main.ts"
+  },
+  "tasks": {
+    "check": "deno check --unstable-raw-imports src/main.ts",
+    "node:start": "deno run -A --inspect --unstable-raw-imports src/main.ts",
+    "dev": "deno task -f @paimaexample/tui clean && NODE_ENV=development PAIMA_API_PORT=9999 deno run -A --check --unstable-raw-imports scripts/start.ts"
+  },
+  "imports": {
+    "@paimaexample/concise": "jsr:@paimaexample/concise@0.3.108",
+    "@paimaexample/config": "jsr:@paimaexample/config@0.3.108",
+    "@paimaexample/log": "jsr:@paimaexample/log@0.3.108",
+    "@paimaexample/crypto": "jsr:@paimaexample/crypto@0.3.108",
+    "@paimaexample/orchestrator": "jsr:@paimaexample/orchestrator@0.3.108",
+    "@paimaexample/runtime": "jsr:@paimaexample/runtime@0.3.108",
+    "@paimaexample/tui": "jsr:@paimaexample/tui@0.3.108",
+    "@paimaexample/utils": "jsr:@paimaexample/utils@0.3.108",
+    "@paimaexample/sm": "jsr:@paimaexample/sm@0.3.108",
+    "@paimaexample/coroutine": "jsr:@paimaexample/coroutine@0.3.108",
+    "@paimaexample/db": "jsr:@paimaexample/db@0.3.108",
+    "fastify": "npm:fastify@^5.4.0",
+    "pg": "npm:pg@^8.14.0",
+    "@sinclair/typebox": "npm:@sinclair/typebox@^0.34.30",
+    "effection": "npm:effection@^3.5.0",
+    "@paimaexample/explorer": "npm:@paimaexample/explorer@0.3.108",
+    "@midnight-ntwrk/compact-runtime": "npm:@midnight-ntwrk/compact-runtime@^0.9.0",
```

Comment:
midnight imports should not be needed 👀

Reply:
ah yeah. Those were copied over from another template. Removing them 👍